types/testutil/network/a.go

```diff
@@ -0,0 +1,771 @@
+package network
```

Comment:
There is the logic called from x/ modules. I have removed dependency on app/ here.

Reply:
This file shouldn't exist. We should try to put everything that depends on `app/` (e.g. the `NewRegenApp()` function) inside `app/`